docs/1-introduction/1.3.1-unix.md

```diff
@@ -40,6 +40,34 @@ mkdir liatrio_console_tutorial
 cd liatrio_console_tutorial
 ```
 
+## MKDIR and Directories
+
+Directories are folders in which you can organize your files. Infact, the directory you are in right now is a folder which holds all the other folders you need. If we navigate to our home directory with `cd ~` and then run `ls` we will be shown all the sub directories in our home (root) directory. We can create a directory on the command line! For those familiar with GUI based folder systems, this is like right clicking and pressing "Create New Folder", however its much simpler on the command line!. To create a directory, we can use the command `mkdir <directory name>`. Lets make a directory like this, `mkdir test_directory`. We can see that we now have a directory in our root `~/test_directory`. Now we an `cd test_directory` to enter into our newely created directory. But lets say we want to create a directory with sub directories within, all from our root (~) directory. Then we can add some cool extensions to this command to make it able to do this. To do this, we can run this command `mkdir -p parent/child/`. The -p flag (thats what these "additions" are called) lets the operating system know that we want the /child directory to be inside of the /parent directory. We can `cd parent` and `ls` and we can see that the /child directory shows up.
```

Comment:
```suggestion
Directories are folders in which you can organize your files. Infact, the directory you are in right now is a folder which holds all the other folders you need. If we navigate to our home directory with `cd ~` and then run `ls` we will be shown all the sub directories in our home directory. We can create a directory on the command line! For those familiar with GUI based folder systems, this is like right clicking and pressing "Create New Folder", however its much simpler on the command line! To create a directory, we can use the command `mkdir <directory name>`. Let's make a directory like this, `mkdir test_directory`. We can see that we now have a directory in our home directory `~/test_directory`. Now we can `cd test_directory` to enter into our newly created directory. Let's say we want to create a directory with sub directories within, all from our home (~) directory. Then we can add some cool flags to this command to make it able to do this. To do this, we can run this command `mkdir -p ~/parent/child/leaf`. The `-p` flag (that's what these "additions" are called) tells the operating system to create any parent directories needed such that `~/parent/child/leaf` will exist at the end (unless there is a name conflict). We can `cd parent` and `ls` and we can see that the `/child` directory shows up.
```

Removed `home (root) directory` because technically "root directory" has a very specific meaning which is `/`. Home directory is more appropriate.

There were a number of spelling errors in this paragraph which makes me wonder if you have ran `npm install` locally? If not please do this so you pull our dependencies. Aside from this allowing you to run the bootcamp locally `npm start` you will get our pre-commit hooks one of which is spell checking.